R/Rarr_utils.R

```diff
@@ -0,0 +1,164 @@
+#' create_zarr_group
+#'
+#' create zarr groups
+#'
+#' @param store the location of (zarr) store
+#' @param name name of the group
+#' @param version zarr version
+#' @export
+create_zarr_group <- function(store, name, version = "v2"){
+  split.name <- strsplit(name, split = "\\/")[[1]]
```

Comment:
It'll be slightly faster if regexes are not used:

```suggestion
  split.name <- strsplit(name, split = "/", fixed = TRUE)[[1]]
```

Reply:
done